### PR TITLE
Ez-959-landing page edit media button hide issue

### PR DESCRIPTION
**Pull request by harishkumar-srijan**
landing page edit media submit button hide issue

### CODE REVIEW

File: pattern-lab/source/_patterns/_patterns-global/scss/admin/layout_builder_modal.scss [hunk 1]
```diff
@@ -322,6 +322,31 @@ ul.paragraphs-add-dialog-list input.field-add-more-submit {
         border: 2px solid $color-primary-hover!important;
       }
     }
+
+    .form-actions {
+      background: transparent;
+
+      .button.button--primary  {
+        display: block!important;
```
Comment (Vinay15) [inline pattern-lab/source/_patterns/_patterns-global/scss/admin/layout_builder_modal.scss:330]: Please explain why we are using !important here.

**Review by Vinay15 — CHANGES_REQUESTED** [review]

File: pattern-lab/source/_patterns/_patterns-global/scss/admin/layout_builder_modal.scss (hunk 1 above)
Comment (harishkumar-srijan) [reply pattern-lab/source/_patterns/_patterns-global/scss/admin/layout_builder_modal.scss:330]: By-default on the inline style is applied on the button to override this need to apply !important here

Comment (Vinay15) [reply pattern-lab/source/_patterns/_patterns-global/scss/admin/layout_builder_modal.scss:330]: Please check why an inline style of `display:none` is getting attached on Save button for a media entity. This should not be the case.

Please check the root cause of the issue and fix accordingly.

**Review by Vinay15 — CHANGES_REQUESTED** [review]

**Review by Vinay15 — APPROVED** [review]
LGTM

**Pull request merged**